yt_dlp/extractor/drtv.py

```diff
@@ -372,3 +373,91 @@ def _real_extract(self, url):
             'formats': formats,
             'is_live': True,
         }
+
+
+class DRTVSeasonIE(InfoExtractor):
+    IE_NAME = 'drtv:season'
+    _VALID_URL = r'''(?x)
+                    https?://
+                        (?:
+                            (?:www\.)?(?:dr\.dk|dr-massive\.com)/drtv/(?:saeson)/
+                        )
+                        (?P<id>[\da-z_-]+)
+                    '''
+    _GEO_COUNTRIES = ['DK']
+    _TESTS = [{
+        'url': 'https://www.dr.dk/drtv/saeson/frank-and-kastaniegaarden_9008',
+        'info_dict': {
+            'id': 'frank-and-kastaniegaarden_9008',
```

Comment:
Isn't it better to use `id=9008`, `display_id=frank-and-kastaniegaarden` ?

Reply:
I didn't see any documentation about this, but sure! I'm not entirely sure whether the number is completely unique though... If so, is that a problem?

Reply:
> I'm not entirely sure whether the number is completely unique though... If so, is that a problem?

For playlists, it's not really an issue